app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+    return count
```

Comment:
You can simplify your code
```suggestion
    return phrase.lower().count(letter.lower())
```

Reply:
My first variant was:
return sum(1 for i in phrase if i.lower() == letter.lower())